arangod/Aql/ClusterNodes.cpp

```diff
@@ -344,13 +324,8 @@ void DistributeNode::toVelocyPackHelper(VPackBuilder& builder, unsigned flags,
   // serialize clients
   writeClientsToVelocyPack(builder);
 
-  builder.add("createKeys", VPackValue(_createKeys));
-  builder.add("allowKeyConversionToObject", VPackValue(_allowKeyConversionToObject));
-  builder.add("fixupGraphInput", VPackValue(_fixupGraphInput));
   builder.add(VPackValue("variable"));
-  _variable->toVelocyPack(builder);
-  builder.add(VPackValue("alternativeVariable"));
-  _alternativeVariable->toVelocyPack(builder);
+  _variable->toVelocyPack(builder);;
```

Comment:
I guess this doesn't break in case of rolling upgrades in the cluster, as coordinators are updated last.
However, it could break an older version of the explainer, which may check any of these attributes on a `DistributeNode`. So we should prepare the explainer in 3.7 to handle the new DistributeNode type gracefully.